### DISCUSSION TITLE
Is pacstall *really* an AUR?

### DISCUSSION THREAD

**Discussion by talwat** [pacstall/pacstall#822] [General]
I've tried submitting a package, but it made me doubt that this is 'The AUR Ubuntu never had.'

The AUR's entire purpose is to be the Arch **User** Repository, but it doesn't seem like it's managed by the users.

If I want to add a package on the AUR, I make a repo, do some fiddling with SSH keys and git, and add it.
There is **zero** involvement from the arch team, because it is the Arch **User** Repository.

However, when I submit a package to pacstall, I need to wait for the maintainers to look at my package, review it, etc...
So is it really an "AUR"? Or just an alternative package manager which is less tested and slower than APT?

**Comment by talwat** [comment]
Well, it seems like the maintainers are **really** fast, so maybe that's why?

**Reply by Elsie19** [reply]
> The AUR's entire purpose is to be the Arch User Repository, but it doesn't seem like it's managed by the users.

It is, we just don't let anyone put random packages into our repos, because we don't use a PKGBUILD format, so people might (usually) get confused as to how things work. If we merged every incoming PR with no questions asked, half the repo wouldn't work.

We have an AUR like *style* to doing things, but we aren't the AUR. We're more of a [nixpkgs](https://github.com/NixOS/nixpkgs) kind of repo.

**Reply by talwat** [reply]
That makes sense, especially since typical distro repos are pretty strict.

**Reply by Elsie19** [reply]
Yeah, we are a user repository, but we have certain standards for packages, if that makes sense, so as long as you adhere to those standards, your package will be merged.
